pkg/apis/config/features.go

```diff
@@ -39,6 +39,15 @@ const (
 	Allowed Flag = "Allowed"
 )
 
+// service annotations under features.knative.dev/*
+const (
+	// PodInfoFeatureKey gates mouting of podinfo with the value 'enabled'
+	PodInfoFeatureKey = "features.knative.dev/mount-podinfo"
```

Comment:
likewise - I'd qualify this 

```suggestion
	QueueProxyPodInfoFeatureKey = "features.knative.dev/queueproxy-podinfo"
```